win/packaging/extra.wxs.in

```diff
@@ -621,7 +606,7 @@
     <CustomAction Id='PresetDatabaseProperties' BinaryKey='wixca.dll' DllEntry='PresetDatabaseProperties' />
     <CustomAction Id="CreateDatabaseCommand" Property="CreateDatabase"
     Value=
-    "&quot;[#F.bin.mysql_install_db.exe]&quot;  &quot;--service=[SERVICENAME]&quot; --port=[PORT] --innodb-page-size=[PAGESIZE] &quot;--password=[ESCAPEDPASSWORD]&quot; &quot;--datadir=[DATADIR]\&quot; [SKIPNETWORKING] [ALLOWREMOTEROOTACCESS] [DEFAULTUSER] --verbose-bootstrap"
+    "&quot;[#F.bin.mariadb_install_db.exe]&quot;  &quot;--service=[SERVICENAME]&quot; --port=[PORT] --innodb-page-size=[PAGESIZE] &quot;--password=[ESCAPEDPASSWORD]&quot; &quot;--datadir=[DATADIR]\&quot; [SKIPNETWORKING] [ALLOWREMOTEROOTACCESS] [DEFAULTUSER] --verbose-bootstrap"
```

Comment:
why?

Reply:
This is out of scope of the current task. As you do not change Linux installers to use <whatever-the-new-name-for-mysql_install_db>, I would not do that here either.

Reply:
This was from before the duplication solution.

Reply:
I'll revert.

Reply:
Reverted in bd5310ad45dfbcec57f696b4ada04083786f94ec

Reply:
Reverted in bd5310ad45dfbcec57f696b4ada04083786f94ec